CLIENT_GENERATOR_GUIDE.md

```diff
@@ -41,20 +41,19 @@ def search(self, index=None, body=None):
     return self.perform_request(method, path, body=body)
 ```
 
-
 ## Overloaded Name
-You will also encounter `x-overloaded-param: "metric"` for the `node_id` path parameter of `GET /_nodes/{node_id}` operation in `nodes.info` action. This is a special case where the path parameter is overloaded to accept either a node ID or a metric name. The `client.nodes.info` method when called with either `metric` or `node_id` (but not both), will use `GET /_nodes/{node_id}` operation (even though the path parameter name is `node_id`). When called with both `metric` and `node_id`, it will use `GET /_nodes/{node_id}/{metric}` operation.
+You will also encounter `x-overloaded-param: metric` for the `node_id` path parameter of `GET /_nodes/{node_id}` operation in `nodes.info` action. This is a special case where the path parameter is overloaded to accept either a node ID or a metric name. The `client.nodes.info` method when called with either `metric` or `node_id` (but not both), will use `GET /_nodes/{node_id}` operation (even though the path parameter name is `node_id`). When called with both `metric` and `node_id`, it will use `GET /_nodes/{node_id}/{metric}` operation.
 
 ## Handling Bulk Operations
-Some operations accept a bulk of data in the request body. For example, the `bulk` action accepts a bulk of index, update, and delete operations on multiple documents. Unlike other operations where the request body is a JSON object, the request body for bulk operations is a newline-seperated JSON string. The client will automatically convert the request body into a newline-seperated JSON objects. The request body of such operations will be denoted with `x-serialize: "bulk"` vendor extension.
+Some operations accept a bulk of data in the request body. For example, the `bulk` action accepts a bulk of index, update, and delete operations on multiple documents. Unlike other operations where the request body is **a JSON object**, the request body for bulk operations is **an array of JSON objects**. When encountering this type of operation, the client must serialize the request body as new-line-seperated strings of JSON objects.
```

Comment:
Typo: seperated -> separated, but the correct terminology is actually that the body is JSON, and for bulk it's newline-delimited JSON, which is a well-known format (https://github.com/ndjson/ndjson-spec).